app/components/works/description_component.html.erb

```diff
@@ -1,4 +1,4 @@
-<section id="description">
+<section id="description" data-controller="nested-form" data-nested-form-select-value=".inner-container">
```

Comment:
I'm pretty sure this `data-nested-form-select-value=".inner-container"` isn't doing anything and can be removed.

Reply:
Why are you changing this component at all?

Reply:
Cruft. I think this was when I was trying to figure out why the changes weren't taking effect. Will remove.